zingoconfig/src/lib.rs

```diff
@@ -70,21 +83,63 @@ pub struct ZingoConfig {
     /// The filename of the logfile. This will be created in the `wallet_dir`.
     pub logfile_name: PathBuf,
 }
-
-impl ZingoConfig {
-    // Create an unconnected (to any server) config to test for local wallet etc...
-    pub fn create_unconnected(chain: ChainType, dir: Option<PathBuf>) -> ZingoConfig {
+impl ZingoConfigBuilder {
+    pub fn set_wallet_dir(mut self, dir: PathBuf) -> Self {
+        self.wallet_dir = Some(dir);
+        self
+    }
+    pub fn set_lightwalletd(mut self, lightwalletd_uri: http::Uri) -> Self {
+        self.lightwalletd_uri = Some(lightwalletd_uri);
```

Comment:
this method is extra steps on top of writing
`zingo_config.lightwalletd_uri =  Arc::new(RwLock::new(Some(lightwalletd_uri)));`